src/ucp/core/ucp_mm.c

```diff
@@ -296,6 +368,19 @@ ucs_status_t ucp_mem_unmap(ucp_context_h context, ucp_mem_h memh)
     return status;
 }
 
+ucs_status_t ucp_mem_query(const ucp_mem_h memh, ucp_mem_attr_t *attr)
+{
+    if (memh->address) {
+        attr->field_mask = UCP_MEM_ATTR_FIELD_ADDRESS |
```

Comment:
i.e. attr should be in/out parameter?
This looks more flexible but requires a bit more actions from user.
And what the function should return if requested field, for example, is not available? `UCS_ERR_UNSUPPORTED`?